plugins/go-build/bin/goenv-install

```diff
@@ -62,6 +62,10 @@ definitions() {
   go-build --definitions | $(type -p ggrep grep | head -1) -F "$query" || true
 }
 
+latest_version() {
+  definitions | grep -oE "^$1\\.?([0-9]+|beta[0-9]+|rc[0-9]+)?" | tail -1
```

Comment:
Just double checking, as I'm not sure I understand the regexp (my bad): are we allowing as "latest" also _beta_ and _rc_ versions?

If so, I'd advice against it: I think it's a much less risky default to assume that "latest" means "latest stable". Who uses _beta_ or _rc_ versions is not likely to be a normal Go developer, trying to go about their business.

Reply:
Your assumption is correct; we can remove RC and Beta and add an "unstable" feature

Reply:
Added `unstable` feature and latest only does latest stable